hack/install-kubestellar.sh

```diff
@@ -16,14 +16,15 @@
 
 # Usage: $0 --create-folder --verbose
 
-# This script installs Kubestellar binaries to a folder of choice
+# This script installs KubeStellar binaries to a folder of choice
```

Comment:
What is the line that says "Usage: $0 --create-folder --verbose" for?  It omits the vast majority of the options.

Reply:
same as above